src/main/java/io/github/jmh07/scoutingeyes/ScoutingEyes.java

```diff
@@ -59,7 +61,7 @@ public void onDisable() {
     public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
 
         boolean isPlayer = sender instanceof Player;
-        boolean hasTwoArgs = args.length == 2;
+        boolean hasTwoArgs = args.length == 2 || args.length == 3;
```

Comment:
nitpick: Consider changing the variable name `hasTwoArgs` to something more appropriate due to the added length condition

Reply:
It could be a good chance and I was too lazy at the time to do it :) . I won't be able to change it for a few day though since I don't have an access to a pc. So you might want to merge and do this hotfix on master.